### PR TITLE
change the extension name

### PR DESCRIPTION
**Pull request by SriKrishnaPaparaju**

### CODE REVIEW

**Comment by SriKrishnaPaparaju** [comment]
@invincibleJai can you please review these changes related to 'name change for the VSCode extension'

File: README.md [hunk 1]
```diff
@@ -37,33 +37,39 @@ Extending support for Python and Go languages is currently under progress.
...
-> Open or edit a manifest file (`pom.xml` / `package.json`) to flag any CVEs present in your application and right click on a manifest file (`pom.xml`/`package.json`) in the 'Vscode File explorer' or  'Vscode File editor' to display a detailed report for your application.
-
+> - Open or edit a manifest file (`pom.xml` / `package.json`) to flag any CVEs present in your application
+  - Right click on a manifest file (`pom.xml`/`package.json`) in the 'Vscode File explorer' or  'Vscode File editor' to display 'Application stack analysis report' for your application.
 
 
 ## Features
 
- Detailed report for your application covering CVEs, license issues, and AI based insights provided on the application dependencies used - flagging dependencies that are rarely used together and suggesting similar alternatives that are more commonly used:
+Application stack analysis report with Insights about your application dependencies:
```
Comment (invincibleJai) [inline README.md]: This information is repeated maybe we can revisit it later

**Comment by invincibleJai** [comment]
Thanks @SriKrishnaPaparaju looks good, just one observation given in the comment.

**Review by precha1989 — COMMENTED** [review]
Duplicate of #

**Pull request merged**